digits/templates/layout.html

```diff
@@ -51,7 +51,8 @@
                 {% if server_name %}
                 <li><span class="navbar-text">Server: {{server_name}}</span></li>
                 {% endif %}
-                <li><span class="navbar-text">Version: {{server_version}}</span></li>
+                <li><span class="navbar-text">Server ver.: {{server_version}}</span></li>
+				<li><span class="navbar-text">Caffe ver.: {{caffe_version}}</span><li>
```

Comment:
^ I meant the whitespace in the code.

But also, it looks kinda cramped and unreadable right now. How about this instead?

![caffe-version-ui](https://cloud.githubusercontent.com/assets/687269/15440848/85bf5ce4-1e8c-11e6-814d-b00eb5b528e8.jpg)